python/tvm/relay/op/transform.py

```diff
@@ -917,7 +922,7 @@ def strided_slice(data, begin, end, strides=None, slice_mode="end"):
         begin = _make.where(begin < cast_like(const(0), begin), begin + ishape_slice, begin)
         begin = _make.where(begin >= ishape_slice, ishape_slice, begin)
         return _dyn_make.strided_slice(data, begin, end, strides, slice_mode)
```

Comment:
Do we not support axes with dynamic begin?

Reply:
Perhaps we should throw an error if we hit this codepath and axes is defined? With a TODO for extending axes to the dynamic case?

Reply:
Yeah I haven't put much thought into `axes` argument for dynamic strided slice. My goal was to preserve more static dimensions along `axes`, which doesn't apply to the dynamic parameter variant. We should support it for convenience and API consistency sake, but for now I'd like to leave it as TODO and assert `axes is None` here until someone complains or comes up with a good use case.

Reply:
Added TODO and assert.